vimeo-networking/src/main/java/com/vimeo/networking/model/FeedItem.java

```diff
@@ -46,44 +49,89 @@
         NONE
     }
 
-    @GsonAdapterKey("uri")
-    public String uri;
-    @GsonAdapterKey("clip")
-    public Video clip;
-    @GsonAdapterKey("type")
-    public String type;
-    @GsonAdapterKey("time")
-    public Date time;
-    @GsonAdapterKey("user")
-    public User user;   // from like type
-    @GsonAdapterKey("channel")
-    public Channel channel; // from channel type
-    @GsonAdapterKey("category")
-    public Category category;
-    @GsonAdapterKey("tag")
-    public Tag tag;
-    @GsonAdapterKey("group")
-    public Group group;
-    @GsonAdapterKey("metadata")
-    public Metadata metadata;
+    @SerializedName("uri")
+    protected String mUri;
+
+    @SerializedName("clip")
+    protected Video mClip;
```

Comment:
In the Android app we need access to set the clip for our wacky object tracking. We will either need to keep these as public or have setters